101-webapp-privateendpoint-vnet-injection/azuredeploy.json

```diff
@@ -0,0 +1,395 @@
+    {
+      "type": "Microsoft.Network/virtualNetworks/subnets",
+      "apiVersion": "2020-04-01",
+      "name": "[concat(parameters('virtualNetwork_name'),'/', parameters('subnet1_name'))]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Network/virtualNetworks', parameters('virtualNetwork_name'))]"
+      ],
+      "properties": {
+        "addressPrefix": "[parameters('subnet1_CIDR')]",
+        "privateEndpointNetworkPolicies": "Disabled"
+      }
+    },
+    {
+      "type": "Microsoft.Network/virtualNetworks/subnets",
+      "apiVersion": "2020-04-01",
+      "name": "[concat(parameters('virtualNetwork_name'),'/', parameters('subnet2_name'))]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Network/virtualNetworks', parameters('virtualNetwork_name'))]",
+        "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('virtualNetwork_name'), parameters('subnet1_name'))]"
+      ],
+      "properties": {
+        "addressPrefix": "[parameters('subnet2_CIDR')]",
+        "delegations": [
+          {
+            "name": "delegation",
+            "properties": {
+              "serviceName": "Microsoft.Web/serverfarms"
+            }
+          }
+        ],
+        "privateEndpointNetworkPolicies": "Enabled"
+      }
+    },
+    {
+      "type": "Microsoft.Web/serverfarms",
+      "apiVersion": "2019-08-01",
+      "name": "[parameters('serverFarm_name')]",
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "[parameters('SKU_name')]",
+        "tier": "[parameters('SKU_tier')]",
+        "size": "[parameters('SKU_size')]",
+        "family": "[parameters('SKU_family')]",
+        "capacity": 1
+      },
+      "kind": "app"
+
+    },
+    {
+      "type": "Microsoft.Web/sites",
+      "apiVersion": "2019-08-01",
+      "name": "[parameters('site1_name')]",
+      "location": "[parameters('location')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Web/serverfarms', parameters('serverFarm_name'))]"
+      ],
+      "kind": "app",
+      "properties": {
+        "enabled": true,
+        "hostNameSslStates": [
+          {
+            "name": "[concat(parameters('site1_name'), parameters('webapp_dns_name'))]",
+            "sslState": "Disabled",
+            "hostType": "Standard"
+          },
+          {
+            "name": "[concat(parameters('site1_name'), '.scm', parameters('webapp_dns_name'))]",
+            "sslState": "Disabled",
+            "hostType": "Repository"
+          }
+        ],
+        "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', parameters('serverFarm_name'))]"
+      }
+    },
+    {
+      "type": "Microsoft.Web/sites",
+      "apiVersion": "2019-08-01",
+      "name": "[parameters('site2_name')]",
+      "location": "[parameters('location')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Web/serverfarms', parameters('serverFarm_name'))]"
+      ],
+      "kind": "app",
+      "properties": {
+        "enabled": true,

```

Comment:
remove this property

Reply:
done

Reply:
It's the enabled property that needs to be removed - here and the previous.

Reply:
sorry, done